protocol/requests/create_account.go

```diff
@@ -59,6 +59,11 @@ type CreateAccount struct {
 
 	TorrentConfigEnabled *bool
 	TorrentConfigPort    *int
+
+	HTTPEnabled bool   `json:"httpEnabled"`
+	APIModules  string `json:"apiModules"`
+	HTTPHost    string `json:"httpHost"`
+	HTTPPort    int    `json:"httpPort"`
```

Comment:
Maybe group this into a `APIConfig` struct?
So when it's `nil`, it's disabled. Otherwise it will be obvious that all fields `modules, host, port` needs to be set. 
And `httpEnabled` is not needed then.

WDYT?